InvenTree/templates/js/translated/order.js

```diff
@@ -2371,17 +2384,7 @@ function loadPurchaseOrderLineItemTable(table, options={}) {
                     });
                 },
                 footerFormatter: function(data) {
-                    var total = data.map(function(row) {
-                        return +row['purchase_price']*row['quantity'];
-                    }).reduce(function(sum, i) {
-                        return sum + i;
-                    }, 0);
-
-                    var currency = (data.slice(-1)[0] && data.slice(-1)[0].purchase_price_currency) || 'USD';
-
-                    return formatCurrency(total, {
-                        currency: currency
-                    });
+                    return '<div id="sub_total_items_price">';
```

Comment:
@bloemp  couldn't you just use the data field to look up the total price in the data field?

Reply:
At the moment the data field doesn't contain the sub_total fields. Therefore I'm trying to get these fields in the output of the non-api get request. But I haven't won that struggle with Django yet
Also the Style JS check is failing, but I see the same construct in other parts of the file which aren't flagged. Changing the quotes to single ones makes the code fail...

Reply:
The code is now changed significantly so this should no longer be an issue